app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (
+    VaccineError,
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(
+            self,
+            visitor: dict
+    ) -> str | VaccineError | NotWearingMaskError:

```

Comment:
Your function doesn't return VaccineError and  NotWearingMaskError. We shouldn't show possible errors in return type hint
```suggestion
    ) -> str:

```